dask/config.py

```diff
@@ -225,6 +246,17 @@ def ensure_file(
         pass
 
 
+def normalize_key(key):
```

Comment:
Nit: I'd prefer this was located closer to `normalize_nested_keys`, to help readability.

Reply:
Makes sense, will do